docs/Deploying-Pega-on-AKS.md

```diff
@@ -446,6 +446,16 @@ Pega added for this purpose in the [pega-helm-charts](https://github.com/pegasys
 
 Make these changes before you begin deploying Pega Platform using Helm charts.
 
+#### Add Support for providing DB credentials using External Secrets Operator
+
+- Create two files following the Kubernetes documentation for External Secrets Operator [External Secrets Operator](https://external-secrets.io/v0.5.1/) :
```

Comment:
minor wording suggestions across all docs. Please also add it to [Deploying-Pega-on-openshift.md] + [Deploying-Pega-on-PKS.md]>>
To avoid exposing the database password, you can configure a JDBC password in the External Secrets Operator. This requires you to create two files using the Kubernetes documentation, External Secrets Operator [External Secrets Operator](https://external-secrets.io/v0.5.1/) :
  •	An external secret file that specifies the required information to fetch in your secret.
  •	A secret store file that defines how your deployment accesses the External Secrets Operator and identifies the required filepath to the files in your Helm directory.

- Copy the files into the pega-helm-charts/charts/pega/templates directory of your Helm repository.
- Update your Helm repository to the latest using the command, $>helm repo update pega https://pegasystems.github.io/pega-helm-charts
- Update your Pega.yaml file: add the `jdbc.external_secret_name` parameter to refer to the External Secret Operator in which you placed your database password.

Thanks so much for completing this!

Reply:
From the External Secrets Operator documentation , it appears this support is not there on openshift and PKS.
The documentation of Providers doesn't include these two cloud providers.

Reply:
Sorry to be a stickler, but the formatting of the intro paragraph doesn't seem right...what I see is kinda borked and looks odd - no separate bullets:
(Optional) Add Support for providing DB credentials using [External Secrets Operator](https://external-secrets.io/v0.5.1/)
- Create two files following the Kubernetes documentation for External Secrets Operator External Secrets Operator : • An external secret file that specifies what information in your secret to fetch. • A secret store to define access how to access the external and placing the required files in your Helm directory.

It should be:
(Optional) Add Support for providing DB credentials using [External Secrets Operator](https://external-secrets.io/v0.5.1/)
(no bullet)Create two files following the Kubernetes documentation for External Secrets Operator External Secrets Operator:
• An external secret file that specifies what information in your secret to fetch.
• A secret store to define access how to access the external and placing the required files in your Helm directory.

Everything seems looks fine.